README.md

```diff
@@ -1,23 +1,23 @@
+# IMCO CLI / Go Library
 
-# Concerto CLI / Go Library
 [![Go Report Card](http://goreportcard.com/badge/ingrammicro/concerto)](http://goreportcard.com/report/ingrammicro/concerto)
 [![Build Status](https://drone.io/github.com/ingrammicro/concerto/status.png)][cli_build] [![GoDoc](https://godoc.org/github.com/ingrammicro/concerto?status.png)](https://godoc.org/github.com/ingrammicro/concerto)
 [![codecov.io](https://codecov.io/github/ingrammicro/concerto/coverage.svg?branch=master)](https://codecov.io/github/ingrammicro/concerto?branch=master)
-[//]: # ([![Build Status](https://travis-ci.org/ingrammicro/concerto.svg?branch=master)](https://travis-ci.org/ingrammicro/concerto))
+[![Build Status](https://travis-ci.org/ingrammicro/concerto.svg?branch=master)](https://travis-ci.org/ingrammicro/concerto)
 
+Ingram Micro Cloud Orchestrator Command Line Interface (aka IMCO CLI) allows you to interact with IMCO features, and build your own scripts calling IMCO's API.
 
+If you are already using IMCO CLI, and only want to obtain the latest version, download IMCO CLI for:
 
-
-IMCO Cloud Orchestrator Command Line Interface allows you to interact with Concerto features, and build your own scripts calling Concerto's API.
-
-If you are already using Concerto CLI, and only want to obtain the latest version, download Concerto CLI for:
 - [Linux][cli_linux]
 - [OSX][cli_darwin]
 - [Windows][cli_windows]
```

Comment:
The problem with these links is that they redirect to the files in the `binaries` directory of master, but, since our SDLC prevents us from committing directly to `master` and the binary for a release cannot be generated until the stable tag on `master` has been published, `master` can never have the latest stable version... I would stop distinguishing by OS and redirect users to https://github.com/ingrammicro/concerto/releases/latest

I would also remove the binaries directory from the repo.